### ISSUE TITLE
Show shell extension menuitems in main File Explorer context menu instead of overflow menu

### ISSUE THREAD

**Issue by Gitoffthelawn** [WinMerge/winmerge#2516]
It would be great to have the WinMerge shell extension's context menuitems appear in Windows 11 File Explorer's main context menu instead of its overflow menu (accessed via "Show more options").  Would this be possible to implement?

Microsoft's PowerToys File Locksmith and PowerRename do this, and it works nicely.

**Comment by sdottaka** [comment]
WinMerge supports the Windows 11 context menu.
Did you install WinMerge from the installer? Sometimes you need to restart Windows after installation.

![image](https://github.com/user-attachments/assets/3e8cb84f-cdc1-49ef-8fcd-e6b8e48aa769)

**Comment by Gitoffthelawn** [comment]
> WinMerge supports the Windows 11 context menu. Did you install WinMerge from the installer? Sometimes you need to restart Windows after installation.

No, I used [Scoop](https://github.com/ScoopInstaller/Scoop).  This is the Scoop manifest for WinMerge: https://github.com/ScoopInstaller/Extras/blob/master/bucket/winmerge.json
